en/source/intro/4.2_cpp_tutorial_en.md

```diff
@@ -130,10 +130,8 @@ int main(){
 }
 ```
 
-
-### 量子状態のデータの取得
-量子状態を表す \f$2^n\f$ の長さの配列を取得します。
-特にGPUで量子状態を作成したり、大きい \f$n\f$ では非常に重い操作になるので注意してください。
+### Obtain data of quantum state
+The quantum state is expressed as an array of length $2^n$. Note that if the quantum state is formed by GPU, and If the $n$ is large, it can become extremely heavy operation.
```

Comment:
```suggestion
The quantum state is expressed as an array of length $2^n$. Especially when using GPU, note that,  if $n$ is large, it can become very computationally heavy.
```